doc/api/esm.md

```diff
@@ -383,6 +385,10 @@ Conditional exports can also be extended to exports subpaths, for example:
   "exports": {
     ".": "./main.js",
     "./feature": {
+      "development": {
```

Comment:
to clarify - can other things in this object be objects too? or are the rest required to be strings?

Reply:
Yes, object nesting is permitted to any depth.